docs/en/interfaces/cli.md

```diff
@@ -158,6 +158,116 @@ $ clickhouse-client --param_tuple_in_tuple="(10, ('dt', 10))" -q "SELECT * FROM
 $ clickhouse-client --param_tbl="numbers" --param_db="system" --param_col="number" --query "SELECT {col:Identifier} FROM {db:Identifier}.{tbl:Identifier} LIMIT 10"
 ```
 
+## Connection string {#connection_string}
+
+The connection string for clickhouse-client is presented in URI format:
+
+```text
+clickhouse://[user_info@][hosts_and_ports][/dbname][?query_parameters]
+```
+
+where user_info is: `user[:password]`
+and hosts_and_ports is a list of values: `[host][:port],[host][:port]` Port is not mandatory.
+and query_parameters is a list of parameter[=value]: `param_name[=value]&param_name[=value]...` value may not be required for some of the parameters. Parameter names are case sensitive.
+
+Allowed query_parameters keys:
+
+- `secure` or shorthanded `s` - no value. If specified, client will connect to the server over a secure connection (TLS). See `secure` in [command-line-options](#command-line-options)
+
+These examples illustrate valid connection strings for clickhouse-client:
```

Comment:
To remove dupliation, suggest to merge these examples with the ones in l. 215

Reply:
Done